### PR TITLE
[webgpu] Uniform buffer size should include padding

### PR DESCRIPTION
**Pull request by axinging**
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

- - -
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7138)

### CODE REVIEW

**Comment by axinging** [comment]
@qjia7 @xhcao  @haoyunfeix  @gyagp PTAL

**Review by qjia7 — APPROVED** [review]
LGTM, thanks!

File: tfjs-backend-webgpu/src/backend_webgpu.ts [hunk 1]
```diff
@@ -720,12 +721,18 @@ export class WebGPUBackend extends KernelBackend {
...
+      }
       currentOffset = Math.ceil(currentOffset / baseAlignment) * baseAlignment;
       preLength = d.data.length;
       offsets.push(currentOffset);
       currentOffset += d.data.length * 4;
     });
 
+    currentOffset = currentOffset % maxAlignmentOfField === 0 ?
+        currentOffset :
+        Math.ceil(currentOffset / maxAlignmentOfField) * maxAlignmentOfField;
```
Comment (xhcao) [inline tfjs-backend-webgpu/src/backend_webgpu.ts:735]: Why not directly use `Math.ceil(currentOffset / maxAlignmentOfField) * maxAlignmentOfField;`

File: tfjs-backend-webgpu/src/backend_webgpu.ts [hunk 2]
```diff
@@ -688,6 +688,7 @@ export class WebGPUBackend extends KernelBackend {
     let currentOffset = 0;
     let preLength = 0;
     const offsets: number[] = [];
+    let maxAlignmentOfField = 4;
```
Comment (xhcao) [inline tfjs-backend-webgpu/src/backend_webgpu.ts:691]: Why not initialize it to `1` here? If there are int8 or fp16 members, will this code be also incorrect?

**Comment by axinging** [comment]
@xhcao Done, PTAL

**Review by xhcao — APPROVED** [review]
Thanks, lgtm

**Pull request merged**